beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -1141,10 +1141,8 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
         >,
```

Comment:
should probably rename this function `get_blocks_checking_caches` or something along these lines.. perhaps we should come up with a unified way of referring to these two caches.. idk if we'd just call them "early caches" or something..

Reply:
https://github.com/sigp/lighthouse/pull/5138/commits/597e05fda2719caf8dabf24b473254125784977b